bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,54 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
 
 #----nodejs install
-curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 sudo apt-get install -y nodejs
 
-#----alsa install
-sudo apt-get install -y alsa-base alsa-utils
+#----install official requirements
+sudo apt-get install -y alsa-base alsa-utils libasound2-dev
+
+#----install missing pigpio in Raspbian Lite (the command npm install pigpio will be exec by package.json)
+sudo apt-get install pigpio
+
+#----install git and download tjbot
+if [ ! -d "${TJBOT_FOLDER}/recipes/conversation" ]; then
+    sudo apt-get install -y git
+
+    rm -Rf /home/pi/git/tjbot
+    git clone https://github.com/ibmtjbot/tjbot.git /home/pi/git/tjbot
+
+    TJBOT_FOLDER='/home/pi/git/tjbot'
+fi
+
+#----install conversation (to install it will resolve tjbotlib and other dependencies)
+cd $TJBOT_FOLDER
+cd recipes/conversation
+echo "path: $PWD"
+
+sudo npm install --unsafe-perm -g node-gyp
```

Comment:
I have never had to do `sudo npm install --unsafe-perm -g node-gyp` before either. In addition, this command simply installs the `node-gyp` package globally in `/usr/lib/node_modules`. It doesn't "install conversation" in `/home/pi/tjbot/recipes/conversation` which according to the comment on line 49 is the intended purpose. The command `npm install` (WITHOUT sudo and WITHOUT --unsafe-perm) appears to be missing.

> I have seen had similar situations where I needed--unsafe-perm to proceed. I cant remember what was the issue, but I think it was again related to npm install for pigpio.

If this was the case then something else was messed up beforehand. Calling `npm install` in `/home/pi/tjbot/recipes/conversation` should be enough to "install conversation".